### DISCUSSION TITLE
Not working properly after deployment

### DISCUSSION THREAD

**Discussion by karthic-keyan** [d3/d3#3738] [Q&A]
Repo -> https://github.com/karthiktrp/D3-Testing

hosted URL --> https://storied-peony-5ffdfd.netlify.app/

some features are not working properly after the deployment 

if i run in the local its working fine but after hosting some feature is missing

local 
![image](https://github.com/d3/d3/assets/58414583/52129ad3-ebe9-41ed-b3a2-66e0884e90b8)

hosted app
![image](https://github.com/d3/d3/assets/58414583/e2379b08-5a09-4061-b6fa-461b9e82e18c)

**Comment by curran** [comment]
This looks like an issue with your application, not an issue with D3.

What evidence do you have that this is an issue with D3 itself? Are there any runtime errors you see?

**Reply by christopherreay** [reply]
the content of the two images is different, as in, the names of the nodes...

it seems possible that it is working as expected

**Reply by karthic-keyan** [reply]
now i updated the image

**Reply by karthic-keyan** [reply]
this is the hosted link -->  https://storied-peony-5ffdfd.netlify.app/

**Comment by karthic-keyan** [comment]
@curran in local its working fine and i am not getting any runtime issue. but after hosting i am getting this issue

please check my repo . let me know if there is any issue

**Reply by christopherreay** [reply]
so i suggest you make some further efforts to describe what is wrong. what works local? what doesn't work published?

perhaps add some links to where in the source code isn't working properly also
